.github/workflows/feature-serverless-framework-deployment.yml

```diff
@@ -94,7 +94,7 @@ jobs:
 
   integration_aws:
```

Comment:
I changed this to `integration_tests` so that integration tests for all providers can be put under here, please amend during rebase.

Also, please add an integration test for Azure under `src/setup/integration-test/serverless-config_test.go` like AWS and GCR